scripts/create-component/index.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+
+const inquirer = require('inquirer')
+const copyTemplateDir = require('copy-template-dir')
+
+const { getPackages, toPackageName } = require('../utils/packages')
+const { smush, pascalize } = require('../utils/string')
+
+const MINIMUM_REACT_VERSION = '^16.2.0'
+
+const packagesDir = path.resolve(__dirname, '..', '..', 'packages')
+const templateDir = path.resolve(__dirname, 'package-template')
+
+const copyPackageFromTemplateDir = (source, dest, vars) => {
+  return new Promise((resolve, reject) => {
+    copyTemplateDir(source, dest, vars, (err, createdFiles) => {
+      if (err) reject(err)
+      else resolve(createdFiles)
+    })
+  })
+}
+
+async function main() {
+  const packages = await getPackages()
+  const packagesMap = packages.reduce((acc, pkg) => ({
+    ...acc,
+    [pkg.name]: pkg.version
+  }))
+
+  const suggestedDependencies = [
+    'button',
+    'core',
+    'icon',
+    'normalize',
+    'theme',
+    'util'
+  ].map(toPackageName)
+
+  const questions = [
+    {
+      type: 'input',
+      name: 'name',
+      message: 'Component Name',
+      validate: name => (name.length > 0 ? true : 'invalid name')
+    },
+    {
+      type: 'checkbox',
+      name: 'dependencies',
+      message: 'Dependencies',
+      choices: suggestedDependencies.map(name => ({ name })),
+      filter: keys => {
+        const depMap = keys.reduce(
+          (acc, name) => ({ ...acc, [name]: packagesMap[name] }),
+          {}
+        )
+        return JSON.stringify(depMap)
+      }
+    }
+  ]
+
+  const answers = await inquirer.prompt(questions)
+
+  const componentName = pascalize(answers.name)
+  const folderName = smush(answers.name)
+  const packageName = toPackageName(answers.name)
+
+  const folderPath = path.resolve(packagesDir, folderName)
+
+  const vars = {
+    ...answers,
+    componentName,
+    folderName,
+    folderPath,
+    packageName,
+    reactVersion: MINIMUM_REACT_VERSION
```

Comment:
I'm not sure where this is actually used, but as far as DS support goes, this min version is too high.  Min version is 15.?.? somewhere.

Reply:
this isn't actually used anymore. will remove